lib/cartopy/crs.py

```diff
@@ -746,7 +747,6 @@ def quick_vertices_transform(self, vertices, src_crs):
                 mod = np.diff(src_crs.x_limits)[0]
                 bboxes, proj_offset = self._bbox_and_offset(src_crs)
                 x_lim = xs.min(), xs.max()
-                y_lim = ys.min(), ys.max()
                 for poly in bboxes:
```

Comment:
@pp-mo What's happened to y_lim?

Reply:
'y_lim' was not used, so flake8 complained about it, so I got rid of it.

There's a lot of that going on in this PR !